### PR TITLE
Set the server manually

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
thanks @hishamco!

File: samples/StatusCodePagesSample/hosting.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
-  "server": "Microsoft.AspNetCore.Server.Kestrel",
-  "server.urls": "http://localhost:5001/base"
```
Comment (Eilon) [inline samples/StatusCodePagesSample/hosting.json]: This one was using a different port. Does that matter for any tests or anything? @Tratcher do you remember if this was deliberate?

Comment (Tratcher) [inline samples/StatusCodePagesSample/hosting.json]: I don't recall. There shouldn't be anything special about this sample though.

Comment (Eilon) [inline samples/StatusCodePagesSample/hosting.json]: Ok nevermind then.

Comment (JunTaoLuo) [inline samples/StatusCodePagesSample/hosting.json]: I tested the samples before merging and there was no particular reason for that specific port.

**Pull request merged**